### PR TITLE
Created components and used router

### PR DESCRIPTION
**Pull request by alphayowakarindi**
At this juncture, I created Navbar, Books, Book, Categories, and AddBooKForm components and also implemented navigation between books and categories pages using the React Router package.

### CODE REVIEW

File: src/components/Categories.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function Category() {
+  return (
+    <div className="category">
+      <h2>Categories Page</h2>
```
Comment (Davidosky007) [inline src/components/Categories.js]: * As stated in the project requirement `Should display a button "Check status" only.` kindly add the button with the text `Check Status` as its a requirement [Check the requirement here ](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md#:~:text=Should%20display%20a%20button%20%22Check%20status%22%20only.)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @alphayowakarindi 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* Well written PR description 👍 
* No linter errors 👍 
* Good looking readme file 👍 
* Proper use of Gitflow 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Categories.js (hunk 1 above)
Comment (alphayowakarindi) [reply src/components/Categories.js]: Hi @Davidosky007, I've fixed it. Thank you.

**Review by elmar8287 — APPROVED** [review]
Hi, @alphayowakarindi.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- All suggestions are done 👍 

### Optional suggestions

- N.A
-

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**